docs/user_guide/data_types_and_io/structureddataset.md

```diff
@@ -252,3 +252,13 @@ You can run the code locally as follows:
 ```
 
 [flytesnacks]: https://github.com/flyteorg/flytesnacks/tree/master/examples/data_types_and_io/
+
+### The nested typed columns
```

Comment:
Can you move this section above line 254? That will keep the target definition at the bottom of the file, where it will be easier to find and update.

Reply:
certainly!